### ISSUE TITLE
Content Not showing

### ISSUE THREAD

**Issue by Rahulgupta-cdnsol** [lucaszischka/BottomSheet#169]
Hi,
I have integrated this source code where I have tablle view in bottomsheet.

When I try to apply apple property(enableAppleScrollBehavior) on it all the content of bottomsheet got disappeared.

Please let me know how to resolve this issue.

thanks,
Rahul

**Comment by samdogg7** [comment]
Make sure your content is in a `VStack` not a `List` / `ScrollView`; `enableAppleScrollBehavior` wraps the content in it's own scroll view

**Comment by lucaszischka** [comment]
I agree with @samdogg7, this will most likely fix your issue. If not, try to get it working like this:

```swift
var body: some View {
  ScrollView {
    // Your content
  }
}
```

Only if this works, but when coping the content code (without the `ScrollView`!) doesn't, it might be an issue with my implementation.
I will consider your issue as resolved, until you give feedback, that it is still not working. In this case please provide me with minimal reproducible code.
